### DISCUSSION TITLE
How to use remote.js in MF with hash build?

### DISCUSSION THREAD

**Discussion by rexchen94** [umijs/umi#11495] [Q&A]
I need use hash:true, but the remote.js is built with the hash name.How can I set the entry of remote with "https//xxx.com/remote.????.js" in another app?

**Comment by fz6m** [comment]
如果要用 hash 的话，就要手动更新，或者第三方拉取，拉取的来源可以是全局变量，没有缓存的特殊脚本，配置中台等体系化设施。
